src/Symfony/Bridge/Twig/composer.json

```diff
@@ -23,7 +23,7 @@
         "symfony/phpunit-bridge": "~2.7",
         "symfony/asset": "~2.7",
         "symfony/finder": "~2.3",
-        "symfony/form": "~2.7",
+        "symfony/form": "~2.7,>=2.7.1",
```

Comment:
This would be the first use of the caret operator isn't it?
We'd rather stick with what we do elsewhere here, and open an other PR for moving everything to the caret operator (and discuss there if it's too early or not)